android-kotlin/app/src/main/java/com/mongodb/app/TaskActivity.kt

```diff
@@ -19,28 +20,43 @@ import io.realm.RealmConfiguration
 import io.realm.kotlin.where
 import io.realm.mongodb.sync.SyncConfiguration
 
-class ItemsActivity : AppCompatActivity() {
-    private lateinit var recyclerView: RecyclerView
-    private var userRealm: Realm? = null
+class TaskActivity : AppCompatActivity() {
+    private lateinit var userRealm: Realm
     private lateinit var config: RealmConfiguration
+    private lateinit var recyclerView: RecyclerView
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_items)
+        setContentView(R.layout.activity_task)
 
         val toolbar = findViewById<View>(R.id.task_menu) as Toolbar
         setSupportActionBar(toolbar)
-        toolbar.showOverflowMenu()
 
         val fab = findViewById<View>(R.id.floating_action_button) as FloatingActionButton
-        fab.setOnClickListener { (addItem()) }
+        fab.setOnClickListener { (onFabClicked()) }
 
-        recyclerView = findViewById(R.id.items_list)
+        recyclerView = findViewById(R.id.task_list)
+        recyclerView.layoutManager = LinearLayoutManager(this@TaskActivity)
+        recyclerView.adapter = object : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+            override fun onCreateViewHolder(
+                parent: ViewGroup,
+                viewType: Int
+            ): RecyclerView.ViewHolder {
+                TODO("Not yet implemented")
```

Comment:
This might be a bit cleaner. I don't think this method can ever be called, but then we're leaving TODOs in people's code. Another option would be to just `throw` in this method.
```            override fun onCreateViewHolder(
                parent: ViewGroup,
                viewType: Int
            ): RecyclerView.ViewHolder {
                // This can never be called
                return object: RecyclerView.ViewHolder(parent.rootView) {}
            }
```